sonar-dotnet-shared-library/src/main/java/org/sonar/plugins/dotnet/tests/CoverageReportImportSensor.java

```diff
@@ -84,11 +85,20 @@ public void execute(SensorContext context) {
   }
 
   void analyze(SensorContext context, Coverage coverage) {
+
+    LOG.debug("Will analyze coverage with wildcardPatternFileProvider with base dir '{}' and file separator '{}'.", BASE_DIR, File.separator);
+
     coverageAggregator.aggregate(wildcardPatternFileProvider, coverage);
 
     Set<String> coverageFiles = coverage.files();
     FileCountStatistics fileCountStatistics = new FileCountStatistics(coverageFiles.size());
+
+    LOG.debug("Will analyze coverage after aggregate found '{}' coverage files.", coverageFiles.size());
```

Comment:
Does Java error message convention put quotes around numbers?
```suggestion
    LOG.debug("Will analyze coverage after aggregate found {} coverage files.", coverageFiles.size());
```
Same situation is on more places - line 148.

Reply:
It's more of a personal convention of making a difference between what's hardcoded and what's variable. I agree with you that in this case the quotes are kinda useless, they don't add a lot of value.

However, for a logging statement like `dotCover parser: found coverage for line '12', hits '0' when analyzing the path ...` , I would prefer to keep the quotes around the numbers because they are actually just text retrieved from a file and having quotes makes this more obvious

Reply:
however for `Parsed NUnit results - total:200, totalSkipped:9, failures:20, errors:30, execution time:null`, the format makes the numbers easier to read, so I don't add quotes.